hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHA.java

```diff
@@ -383,16 +386,27 @@ protected void testCreateFile(OzoneBucket ozoneBucket, String keyName,
 
     OzoneKeyDetails ozoneKeyDetails = ozoneBucket.getKey(keyName);
 
-    Assert.assertEquals(keyName, ozoneKeyDetails.getName());
-    Assert.assertEquals(ozoneBucket.getName(), ozoneKeyDetails.getBucketName());
-    Assert.assertEquals(ozoneBucket.getVolumeName(),
+    assertEquals(keyName, ozoneKeyDetails.getName());
+    assertEquals(ozoneBucket.getName(), ozoneKeyDetails.getBucketName());
+    assertEquals(ozoneBucket.getVolumeName(),
         ozoneKeyDetails.getVolumeName());
-    Assert.assertEquals(data.length(), ozoneKeyDetails.getDataSize());
+    assertEquals(data.length(), ozoneKeyDetails.getDataSize());
+    assertTrue(ozoneKeyDetails.isFile());
 
     try (OzoneInputStream ozoneInputStream = ozoneBucket.readKey(keyName)) {
       byte[] fileContent = new byte[data.getBytes(UTF_8).length];
       ozoneInputStream.read(fileContent);
-      Assert.assertEquals(data, new String(fileContent, UTF_8));
+      assertEquals(data, new String(fileContent, UTF_8));
+    }
+
+    Iterator<? extends OzoneKey> iterator = ozoneBucket.listKeys("/");
+    while (iterator.hasNext()) {
+      OzoneKey ozoneKey = iterator.next();
+      if (Objects.equals(keyName, ozoneKey.getName())) {
+        assertTrue(ozoneKey.isFile());
+      } else if (keyName.startsWith(ozoneKey.getName())) {
+        assertFalse(ozoneKey.isFile());
+      }
```

Comment:
Is it the same to use `/` in the condition?

```suggestion
      if (!ozoneKey.getName().endsWith(OM_KEY_PREFIX)) {
        assertTrue(ozoneKey.isFile());
      } else {
        assertFalse(ozoneKey.isFile());
      }
```

Reply:
Yes, it is same to use `/`.

Thanks for the suggestion.